amazon/iontest/ion_test_driver.py

```diff
@@ -16,8 +16,8 @@
 
 Usage:
     ion_test_driver.py [--implementation <description>]... [--ion-tests <description>] [--test <type>]...
-                       [--local-only] [--cmake <path>] [--git <path>] [--output-dir <dir>] [--results-file <file>]
-                       [<test_file>]...
+                       [--local-only] [--cmake <path>] [--git <path>] [--maven <path>] [--java <path>]...
```

Comment:
One thing I didn't notice before: there should not be a `...` after `--java` because it cannot be specified more than once.